lib/ansible/modules/packaging/os/yum.py

```diff
@@ -193,6 +194,7 @@
   yum:
     name: '*'
     state: latest
+    exclude: ansible
```

Comment:
Reviewing the code there is a real shortcoming. You can only exclude one single package this way.
This should be redesigned to be a proper YAML list, so we can simply do:
```yaml
  exclude:
  - ansible
  - yum
```

Are you comfortable to make this change in Python too ?